src/components/CommandUtils.ts

```diff
@@ -28,6 +28,7 @@ export const symDf: SymbolDf = {
 
 export type ColEnumArgs = Record<string, string>;
 
+
 export type Atom = number | string | SymbolT | ColEnumArgs;
 export type SettableArg = number | string | ColEnumArgs;
```

Comment:
ColEnumArgs is a special type of spec.  It specifies a dictionary from column name to a strict enum.  It allows different operations to be chosen per column